_studio/shared/umc/codec/mpeg2_dec/src/umc_mpeg2_bitstream.cpp

```diff
@@ -295,6 +296,15 @@ namespace UMC_MPEG2_DECODER
         if (CheckBSLeft())
             throw mpeg2_exception(UMC::UMC_ERR_INVALID_STREAM);
 
+        uint32_t width_in_MBs  = mfx::align2_value<uint32_t>(slice.GetSeqHeader().horizontal_size_value, 16) / 16u;
+        uint32_t height_in_MBs = mfx::align2_value<uint32_t>(slice.GetSeqHeader().vertical_size_value, 16) / 16u;
+
+        // invalid slice - skipping it
+        if (slice.sliceHeader.slice_vertical_position >= height_in_MBs)
```

Comment:
Should this be `if (slice.sliceHeader.slice_vertical_position > height_in_MBs)` instead?  It seems to get rid of the regression I reported.

Reply:
Yep, I confirm that's a bug.